applications/tari_app_utilities/src/identity_management.rs

```diff
@@ -49,7 +49,7 @@ pub fn setup_node_identity<P: AsRef<Path>>(
     peer_features: PeerFeatures,
 ) -> Result<Arc<NodeIdentity>, ExitCodes>
 {
-    match load_identity(identity_file.as_ref().clone()) {
+    match load_identity(<&std::path::Path>::clone(&identity_file.as_ref())) {
```

Comment:
```suggestion
    match load_identity(&identity_file) {
```

Reply:
:+1: I see it's used later - `identity_file.as_ref()` is also acceptable

Reply:
or changing the others below to `&identity_file`